config/locales/en.yml

```diff
@@ -637,6 +637,8 @@ en:
       support_user_responsible_body_form:
         name: Responsible body name
   banners:
+    secondary_mass_testing:
+      heading: All secondary schools closed for mass testing can order laptops and devices now
```

Comment:
Update to `All secondary schools can order laptops and devices now`

Reply:
Text is 'laptops and tablets'